apps/web/app/[locale]/page-component.tsx

```
@@ -119,12 +119,8 @@ function MainPage() {
 							<ResizableHandle withHandle />
 
 							{/* </Container> */}
-							<ResizablePanel
-								defaultSize={65}
-								maxSize={95}
-								className="!overflow-y-scroll custom-scrollbar"
-							>
-								<div>{isTeamMember ? <TeamMembers kanbanView={view} /> : <NoTeam />}</div>
+							<ResizablePanel defaultSize={65} maxSize={95} className="!overflow-y-auto custom-scrollbar">
+								<>{isTeamMember ? <TeamMembers kanbanView={view} /> : <NoTeam />}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)